src/__experimental__/components/decimal/decimal.spec.js

```diff
@@ -1697,4 +1698,14 @@ describe('Decimal', () => {
       }));
     });
   });
+  it('the required prop is passed to the input', () => {
+    render({ label: 'required', required: true });
+
+    const input = wrapper.find('input');
+    expect(input.at(0).prop('required')).toBe(true);
+    expect(input.at(1).prop('required')).toBe(undefined); // hidden input
+
+    const label = wrapper.find(Label);
+    expect(label.prop('isRequired')).toBe(true);
+  });
```

Comment:
two tests in one case

Reply:
Updated test description because the setup is the same.